modules/ssh/ssh.go

```diff
@@ -35,11 +35,11 @@ type Host struct {
 }
 
 type ScpDownloadOptions struct {
-	FileNameFilter string //bash style match string for files
-	MaxFileSizeMB  int    //Don't grab any files > MaxFileSizeMB
-	RemoteDir      string //Copy this directory on the remote machine
-	LocalDir       string //Copy RemoteDir to this directory on the local machine
-	RemoteHost     Host   //Connection information for the remote machine
+	FileNameFilters []string //bash style match string for files
```

Comment:
Is there a better way to word this comment?

Reply:
"File names to match. May include bash-style wildcards. E.g., *.log."

Reply:
thanks!